app/main.py

```diff
@@ -20,7 +24,30 @@
     allow_headers=["*"],
 )
 
+root_admin = User(username="root", email="root@example.com", password="password", isAdmin=True)
+
+def create_root_admin():
+    """
+    Create a root admin user.
+
+    Returns:
+    - A dictionary containing the ID of the newly created user.
+    """
+    collection = db.users
+    existing_user = collection.find_one({"username": "root"})
+    if existing_user:
+        print("Root admin user already exists.")
+    else:
+        root_admin_data = root_admin.dict()
+        root_admin_data["isAdmin"] = True
+        collection.insert_one(root_admin_data)
+
+create_root_admin()
+
```

Comment:
Please move the function definition elsewhere, `main.py` should only contain logic related to project initialization, not definitions, keep it abstract :skull:, just keep the function call since it counts as initialization.